include/jwt-cpp/jwt.h

```diff
@@ -2316,13 +2321,13 @@ namespace jwt {
 		/**
 		 * Attempt to parse JSON was unsuccessful
 		 */
-		struct invalid_json_exception : public std::runtime_error {
+		struct invalid_json_exception : exception, public std::runtime_error {
```

Comment:
See above, given that we dont actually set a dynamic error message, we should probably add a "invalid_json" error code and stick that in.